src/models/Tutor.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require("mongoose");
+
+const TutorSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Types.ObjectId,
+        ref: "User",
+        required: true,
+    },
+    
+    subjectId: {
```

Comment:
"error": "Tutor validation failed: subjectId: Path `subjectId` is required."
 We don't have sibjectId yet, I think, this field must be temporarily commented .

Reply:
done:)